src/main/scala/org/constellation/util/Signature.scala

```diff
@@ -0,0 +1,133 @@
+package org.constellation.util
+
+import java.security.{KeyPair, PublicKey}
+
+import cats.kernel.Monoid
+import constellation._
+import org.constellation.crypto.KeyUtils
+import org.constellation.crypto.KeyUtils._
+import org.constellation.primitives.{Edge, Schema, Transaction}
+import org.constellation.primitives.Schema._
+
+
+trait Signable extends AnyRef {
```

Comment:
does extends AnyRef do anything? All objects implicitly extend from AnyRef. technically doesn't hurt to have it but you can remove that.

Reply:
Oh was just ensuring the kryo argument was met, yeah I guess this is redundant